### DISCUSSION TITLE
Is it possible to do this?

### DISCUSSION THREAD

**Discussion by MaxLiu0325** [react-navigation/react-navigation#10665] [Q&A]
Is it possible to do this?

![RPReplay_Final1656385815](https://user-images.githubusercontent.com/33539383/176084376-ce46a23f-93cd-42bd-96cb-a2d5b9323996.gif)

**Comment by kacperkapusciak** [comment]
Hello @MaxLiu0325 :wave:

The pattern that can be seen in the top-right corner of your recording is called 'Shared Element Transition' and as far as I know, it isn't officially supported with `react-navigation`. However, you could try using [one of the community solutions](https://github.com/IjzerenHein/react-navigation-shared-element) for this.

Cheers
